tests/test_models/test_editors/test_fastcomposer/test_fastcomposer.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import gc
+from unittest import TestCase
+
+import numpy as np
+import torch
+from PIL import Image
+
+from mmagic.registry import MODELS
+from mmagic.utils import register_all_modules
+
+gc.collect()
+torch.cuda.empty_cache()
+
+register_all_modules()
+stable_diffusion_v15_url = 'runwayml/stable-diffusion-v1-5'
+clip_vit_url = 'openai/clip-vit-large-patch14'
+finetuned_model_path = ''
+
+config = dict(
+    type='FastComposer',
+    vae=dict(type='AutoencoderKL', sample_size=64),
+    unet=dict(
+        sample_size=64,
+        type='UNet2DConditionModel',
+        down_block_types=('DownBlock2D', ),
+        up_block_types=('UpBlock2D', ),
+        block_out_channels=(32, ),
+        cross_attention_dim=16,
+    ),
+    text_encoder=dict(
+        type='ClipWrapper',
+        clip_type='huggingface',
+        pretrained_model_name_or_path=stable_diffusion_v15_url,
+        subfolder='text_encoder'),
+    tokenizer=stable_diffusion_v15_url,
+    pretrained_cfg=dict(
+        finetuned_model_path=finetuned_model_path,
+        enable_xformers_memory_efficient_attention=None,
+        pretrained_model_name_or_path=stable_diffusion_v15_url,
+        image_encoder_name_or_path=clip_vit_url,
+        revision=None,
+        non_ema_revision=None,
+        object_localization=None,
+        object_localization_weight=0.01,
+        localization_layers=5,
+        mask_loss=None,
+        mask_loss_prob=0.5,
+        object_localization_threshold=1.0,
+        object_localization_normalize=None,
+        no_object_augmentation=True,
+        object_resolution=64),
+    scheduler=dict(
+        type='DDPMScheduler',
+        from_pretrained=stable_diffusion_v15_url,
+        subfolder='scheduler'),
+    test_scheduler=dict(
+        type='DDIMScheduler',
+        from_pretrained=stable_diffusion_v15_url,
+        subfolder='scheduler'),
+    dtype='fp32',
+    data_preprocessor=dict(type='DataPreprocessor'))
+
+
```

Comment:
@pytest.mark.skipif(
    'win' in platform.system().lower(),
    reason='skip on windows due to limited RAM.')

Looks like there is a windows CI bug which complains not enough memory.
You can add this to the head of this class to avoid it.

Reply:
for example, see tests/test_models/test_editors/test_deblurganv2/test_deblurganv2.py

Reply:
OK, thanks